main.go

```diff
@@ -182,44 +184,21 @@ func loop() {
 	if *hibernate {
 		return
 	}
-	// autoextract self
-	src, _ := os.Executable()
-	dest := filepath.Dir(src)
-
-	if embeddedAutoextract {
-		// save the config.ini (if it exists)
-		if _, err := os.Stat(filepath.Join(dest, "config.ini")); os.IsNotExist(err) {
-			log.Println("First run, unzipping self")
-			err := utilities.Unzip(src, dest)
-			log.Println("Self extraction, err:", err)
-		}
-	}
 
-	// Parse ini config
-	args, err := parseIni(filepath.Join(dest, "config.ini"))
-	if err != nil {
-		panic(err)
-	}
-	err = iniConf.Parse(args)
-	if err != nil {
-		panic(err)
-	}
+	log.SetLevel(log.InfoLevel)
+	log.SetOutput(os.Stdout)
 
-	// Parse additional ini config
-	args, err = parseIni(filepath.Join(dest, *additionalConfig))
-	if err != nil {
-		panic(err)
-	}
-	err = iniConf.Parse(args)
-	if err != nil {
-		panic(err)
-	}
+	// the important folders of the agent
+	src, _ := os.Executable()
+	srcPath := paths.New(src)  // The path of the agent's binary
+	srcDir := srcPath.Parent() // The directory of the agent's binary
+	usr, _ := user.Current()
```

Comment:
I don't know if `user.Current()` returns what we expect on Windows, we should copy it from the CLI